bundles/Billing/Controller/ApiController.php

```diff
@@ -241,7 +242,13 @@ public function getProjectWorklogs(BillingService $billingService, $projectId)
      */
     public function recordInvoice(BillingService $billingService, $invoiceId)
     {
-        return new JsonResponse($billingService->recordInvoice($invoiceId));
+        try {
+            $invoice = $billingService->recordInvoice($invoiceId);
+
+            return new JsonResponse($invoice);
+        } catch (Exception $e) {
```

Comment:
This is a very general exception to catch. Do we risk hiding exceptions that are not directly related to the actual service (api) call?

Reply:
Change to custom exception